packages/UI/CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# Getting started with the Platform Enablement Team
+
+dev-launchers-platform is a mono repo using yarn. All yarn script lines need to be run in the root folder. 
+
+## Installing and quick started
+1. install yarn, below are several ways to install. 
+    - https://classic.yarnpkg.com/en/docs/install#windows-stable
+    - In a terminal with admin run ```$ corepack enable```
+    - ```$ npm install --global yarn```
+2. clone the repo @  https://github.com/dev-launchers/dev-launchers-platform
+3. After cloning repo, navigate to /dev-launchers-platform
+4. ```$ yarn install``` in the root folder to install dependencies 
+5. Open a new terminal and run ```$ yarn workspace @devlaunchers/tailwind dev```. to start tailwind constructor.
+5. Open a 2nd terminal and run ```$ yarn workspace @devlaunchers/components storybook``` to start storybook.
+7. Start Development work.
+
+## Yarn scripts basics.
+
+This is just a quick rundown on how to run scripts in a mono repo using yarn, you will need to target a workspace with the following CLI.  Further information can be found in the root folder markdowns. 
+
+    $ yarn workspace @devlaunchers/<workspace> <cmd>
+
```

Comment:
this is general to yarn and its usage so we remove it from here and maybe in the future add it to a CONTRIBUTING.md file in the root of the monorepo